mmv1/products/gkehub2/MembershipRBACRoleBinding.yaml

```diff
@@ -0,0 +1,161 @@
+    test_env_vars:
+      project: :PROJECT_NAME
+      location: 'global'
+# Skip sweeper gen since this is a child resource.
+skip_sweeper: true
+id_format: 'projects/{{project}}/locations/{{location}}/memberships/{{membership_id}}/rbacrolebindings/{{membership_rbac_role_binding_id}}'
+import_format:
+  ['projects/{{project}}/locations/{{location}}/memberships/{{membership_id}}/rbacrolebindings/{{membership_rbac_role_binding_id}}']
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'membership_id'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Id of the membership
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Location of the Membership
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'membershipRbacRoleBindingId'
+    description: |
+      The client-provided identifier of the RBAC Role Binding.
+    required: true
+    immutable: true
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    description: |
+      The resource name for the RBAC Role Binding
+  - !ruby/object:Api::Type::String
+    name: 'uid'
+    output: true
+    description: |
+      Google-generated UUID for this resource.
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the RBAC Role Binding was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the RBAC Role Binding was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'deleteTime'
+    description: |
+      Time the RBAC Role Binding was deleted in UTC.
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'state'
+    description: |
+      State of the RBAC Role Binding resource.
+    output: true
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'code'
+        description: |
+          Code describes the state of a RBAC Role Binding resource.
+        output: true
+        values:
+          - :CODE_UNSPECIFIED
+          - :CREATING
+          - :READY
+          - :DELETING
+          - :UPDATING
+  - !ruby/object:Api::Type::String
+    name: 'user'
+    description: |
+      Principal that is be authorized in the cluster (at least of one the oneof
+      is required). Updating one will unset the other automatically.
+      user is the name of the user as seen by the kubernetes cluster, example
+      "alice" or "alice@domain.tld"
+    exactly_one_of:
+      - user
+      - group
+  - !ruby/object:Api::Type::String
+    name: 'group'
+    description: |
+      Principal that is be authorized in the cluster (at least of one the oneof
+      is required). Updating one will unset the other automatically.
+      group is the group, as seen by the kubernetes cluster.
+    exactly_one_of:
+      - user
+      - group
```

Comment:
Could you set this value in a test?

Reply:
Removing this field; recently found out its not usable in the API.